packages/engine/lib/memory/Cargo.toml

```diff
@@ -21,6 +21,7 @@ tracing = "0.1.35"
 uuid = { version = "1.1.2", features = ["v4"] }
 num = "0.4.0"
 bytemuck = "1.10.0"
+lazy_static = "1.4.0"
```

Comment:
We can use `Lazy` from `std` instead

Reply:
The structs in `std::lazy` were moved, It's called [`std::sync::LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) now.